hmt_escrow/job.py

```diff
@@ -1090,17 +1110,19 @@ def _create_escrow(self, gas: int = GAS_LIMIT) -> bool:
 
         Raises:
             TimeoutError: if wait_on_transaction times out.
-
+        
         """
+
         txn_func = self.factory_contract.functions.createEscrow
+        func_args = []
         txn_info = {
             "gas_payer": self.gas_payer,
             "gas_payer_priv": self.gas_payer_priv,
             "gas": gas
         }
-
-        handle_transaction(txn_func, *[], **txn_info)
-        return True
+        handle_transaction(txn_func, *func_args, **txn_info)
```

Comment:
This change makes sense. This however might be a quite big breaking change in itself. Make sure to check this out in all the doctests as we might be using this quit a lot (`._create_escrow()`)

Reply:
Let's stick w/ previous style for now. Race conditions on getting the last address could cause problems